Add Crypto Legions Bloodstone (BEP20)

abu7shish opened the pull request:
Adding info for BEP20 token 'Crypto Legions Bloodstone'.



PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

cryptomanz1 commented:
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

abu7shish commented:
• What is the purpose of this token? How will it benefit Trust Wallet users?

https://docs.cryptolegions.app/
Our ambition is to create the longest lasting play-to-earn NFT game on the BSC network with a lucrative ROI for players.

1. Crypto Legions has Solved the Longevity Problem for Play-To-Earn NFT Games
2. We invented the Reserve Pool™
3. A 100% completed and tested game before launch

The first P2E NFT Game with an Omni-Balanced Oracle™
While other play-to-earn games regularly suffer economic catastrophe, Crypto Legions is protected against their tragic fate. Nadodo, our revolutionary Omni-Balanced Oracle™, is ever present, ensuring a thriving, sustainable economy for long-term players.
Like a sentinel keeping watch in the night while the villagers sleep, or a hunter adjusting his aim based on the speed of his prey, Nadodo deploys a carefully calibrated approach to eliminate threats to the game economy before players even notice them.
Nadodo is similar to a bulletproof vest for the economy. You don't feel much difference when having one, but it can save your life.
Our economy system has been reviewed by a PhD professor in Gaming and Mathematics at one of the top five universities in the UK. Every day, the economic situation is being reviewed by at least one economist, and Nadodo will regularly make the necessary subtle adjustments for a sustainable economy.
Our Omni-Balanced Oracle™ System contains amongst others the following elements:
$BLOODSTONE rewards when hunting monsters are pegged to the USD currency (you will always have the same USD reward for the same monsters)
The rewards you earned from hunts, that are in your unclaimed wallet, preserve their value, because they are pegged to the USD
A sliding adjustable marketplace tax, between 5% to 20%. Higher taxes if the Reward Pool needs replenishment, lower tax if the Rewards Pool grows.
A sliding adjustable hunting tax, between 0% to 9%. Every time one goes hunting, they pay a fee in BUSD that goes straight to the Liquidity Pool. The amount of tax depends whether the Liquidity Pool needs replenishment or not. (We have updated the destination of the hunting tax. [See the update of 2022-07-19 here.](https://github.com/tokenomics))
A sliding adjustable buy tax (range 0% - 2%), going to Liquidity Pool.
A sliding adjustable sell tax (range 0% - 20%), including adjustable percentages for each of the items inside the sell tax (see table below for all details).
A sliding adjustable damage for legions for each hunt, between 1% to 3%.
A sliding adjustable summoning fee, between 9 USD to 40 USD. 
A sliding adjustable discount fee for hunting supplies for 14 and 28 days (see table below for all details).
The adjustable taxes are reviewed and adjusted accordingly, if needed, about twice per week, or more often in case of emergencies.
These player-rewarding taxes serve to provide ample rewards for players, and create a sustainable economy to preserve the longevity of the game.
Nadodo is monitoring the economic situation every day, and intends to implement pro-actively gentle changes to the adjustable variables, so that players feel minimal changes. In the table below, you can see the minimum and maximum values that Nadodo is allowed to implement. If Nadodo sets a value above the usual average, then it incentivizes the player rewards (reward pool, liquidity pool, etc). If Nadodo sets a value below the usual average, then it incentivizes the players directly.

• How is the token being distributed? Are you conducting an airdrop?

Token can be acquired by users on dex exchanges, and later on on cex.

• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
https://www.pinksale.finance/launchpad/0x4bB0A2Df3b0F25545bA6301FD44b801A094918b8?chain=BSC

• Is the token listed on a centralized/decentralized exchange?

Yes it is traded on Pancakeswap, current list is available here https://coinmarketcap.com/currencies/crypto-legions-bloodstone/

• Is a Smart Contract audit completed? Where can we find this information?

Yes. The token is audited by Certik, you can find information publicly available here - https://www.certik.com/projects/crypto-legions
And the team is already KYC by Solidproof, you can find information publicly available here - https://github.com/solidproof/projects/blob/main/Crypto%20Legions/KYC_Certificate_Crypto_Legions.png
